app/components/jobseekers/search_results/job_alerts_link_component.html.slim

```diff
@@ -3,4 +3,4 @@
   .job-alert-cta__content class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-0" id="job-alert-cta"
     .icon.icon--left.icon--alert-white
       = link_to t("subscriptions.link.text"), new_subscription_path(search_criteria: @vacancies_search.only_active_to_hash), id: "job-alert-link-sticky-gtm"
-      = t("subscriptions.link.help_text")
+      =< t("subscriptions.link.help_text")
```

Comment:
What about passing the link into the translation instead? I think it's cleaner that way